src/com/facebook/buck/python/PythonBuckConfig.java

```diff
@@ -286,6 +301,7 @@ public PackageStyle getPackageStyle() {
   public enum PackageStyle {
     STANDALONE,
     INPLACE,
+    PEX_INPLACE,
```

Comment:
Hmm I think if the goal is to remove most of the pex-specific parts of the code, I think just adding something like `python:pex_args` and using the existing python platform flavor machinery might be the right approach. At that point I think it would make sense to convert the `INPLACE` style to this scheme, by e.g. having a magic `path_to_pex` value that uses the existing inplace code. Alternatively, an option might be to extract the existing inplace code into a java binary, although that might be a bit more of a rewrite.